tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py

```diff
@@ -187,4 +188,43 @@ def test004_account_add_delete_user(self):
         self.assertNotIn('%s@itsyouonline' % self.vdcuser,
                          [user['userGroupId'] for user in account['acl']])
 
-        self.log('%s ENDED' % self._testID)
+
+    @unittest.skip('https://github.com/openvcloud/0-templates/issues/95')
+    def test005_account_add_delete_non_existing_user(self):
+        """ ZRT-OVC-021
+        *Test case for add and delete  non-existing user from account. *
+
+        **Test Scenario:**
+
+        #. Create an account (A1).
+        #. Add a non-existing user to A1, should fail.
+        #. Delete a non-existing user from A1, should fail.
+        """
+        self.log('%s STARTED' % self._testID)
+
+        self.accounts[self.acc1] = {'openvcloud': self.openvcloud}
+        self.log('Create an account, should succeed')
+        res = self.create_account(openvcloud=self.openvcloud, vdcusers=self.vdcusers,
```

Comment:
Both openvcloud and vdcusers don't have to be passed here

Reply:
no , as we didn't put them on create account method